traffic_ops/traffic_ops_golang/federation_resolvers/federation_resolvers.go

```diff
@@ -136,13 +135,28 @@ func Read(w http.ResponseWriter, r *http.Request) {
 	} else {
 		log.Warnf("Couldn't get config %v", e)
 	}
+
+	// Based on version we load types - for version 5 and above we use FederationResolverV5
+	var resolvers []interface{}
+	if inf.Version != nil && inf.Version.Major >= 5 && inf.Version.Minor >= 0 {
+		frData := []tc.FederationResolverV5{} // FR type data
+		for _, item := range frData {
+			resolvers = append(resolvers, item)
+		}
+	} else {
+		frData := []tc.FederationResolver{} // PR type data
+		for _, item := range frData {
+			resolvers = append(resolvers, item)
+		}
+	}
```

Comment:
This entire code block is not needed, because, on lines 142 and 147, you're declaring a new variable (array), and on the subsequent lines, you're traversing over that array. There are 0 entries in the array, so traversing through it is needless.